assets/scss/base/form.scss

```diff
@@ -1,32 +1,9 @@
 form {
-  input[type='text'],
-  input[type='number'] {
-    color: currentColor;
-    border: 1px solid currentColor;
-    font-size: 1.25rem;
-    font-weight: 400;
-    border-radius: 3px;
-
-    @include media-breakpoint-down(md) {
-      font-size: 1.5rem;
-    }
-
-    &:focus {
```

Comment:
@screeper The Vendor Detail view is a bit wonky with those changes:

* Focus Outline with the thick borders and hard shadows looks a bit out of place.
* The Input Size is no longer aligned with the button size.

It porbably makes sense to remove those styles globally, but in that specific place they make sense.

Careful when fixing: The sizes have to be correct in multiple breakpoints.